exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+   "book-store": {
+      "description": "Calculate lowest price for shopping basket only containing books from a single series",
+      "cases": [
+         {
+            "description": "A basket containing only a single book is charged full price.",
+            "basket": [1],
+            "expected": 8.0
+         },
+         {
+	    "description": "A basket containing only two of the same book is charged full price.",
```

Comment:
sorry, mind making this align? there's one space missing, compared to "basket" below.